packages/ember-routing-htmlbars/lib/keywords/closure-action.js

```diff
@@ -92,7 +92,11 @@ function createClosureAction(stream, target, action, valuePath, actionArguments)
 
       let payload = { target, args, label: labelFor(stream) };
       return flaggedInstrument('interaction.ember-action', payload, () => {
-        return run.join(target, action, ...args);
+        let params = [target, action].concat(args);
```

Comment:
In theory yes, not sure if that will still pacify Chakra, but I'll give it a shot.